docs/best-practices/testing/how-to-use-hypernode-docker.md

```diff
@@ -0,0 +1,145 @@
+---
+myst:
+  html_meta:
+    description: 'Docker image can be used to set up a fast and easy local development
+      environment for Hypernode. See intstructions that guide you through the process. '
+    title: How to use hypernode docker?
+redirect_from:
+  - /en/hypernode/hypernode-docker/how-to-use-hypernode-docker/
```

Comment:
Needs to be `/hypernode-docker` instead, because that's the URL on https://community.hypernode.io. Please apply this to the rest of the pages as well